### ISSUE TITLE
csse_covid_19_daily_reports/03-13-2020.csv wrong date (11.  instead of 13.) 

### ISSUE THREAD

**Issue by at062084** [CSSEGISandData/COVID-19#737]
cat 03-13-2020.csv| grep 11T | grep ^, | sort | less
shows  countries where date is 2020-03-11 instead of 2020-03-13
data are correct though for e.g. Austria, Germany, Iran, ...

**Comment by avengingangels** [comment]
These dates appear to be incorrect, particularly as the have higher case counts for lots of countries than 03-12-2020.csv entries. 

For example, in "03-12-2020.csv" Austria has 302 confirmed cases with the Last Update set as
"2020-03-12T11:13:29".

In "03-13-2020.csv" Austria has 504 confirmed cases with the Last Update set as "2020-03-11T20:00:00".

**Comment by tlmohren** [comment]
Indeed, almost all entries in  "03-13-2020.csv" still have the date to be2020-03-11. I'll correct this post import but it would be great if the file can be corrected.
